src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -5,13 +5,30 @@ import lanternFooter from './images/lantern-footer.png';
 import mapImg from './images/light-up-map.gif';
 import { Trans, useTranslation } from 'react-i18next';
 
+import Modal from '../ModalPayment';
+
+import {
+  useModalPaymentDispatch,
+  ModalPaymentConstants,
+  ModalPaymentTypes,
+} from '../../utilities/hooks/ModalPaymentContext';
+
 const LightUpChinatownPage = () => {
+  const ModalPaymentDispatcher = useModalPaymentDispatch(null);
   const { t } = useTranslation();
   const today = new Date();
   const campaignEndDate = new Date('11/30/2020');
   const timeUntilEnd = campaignEndDate.getTime() - today.getTime();
   const daysUntilEnd = Math.ceil(timeUntilEnd / (1000 * 3600 * 24));
 
+  const openModal = (event) => {
+    event.preventDefault();
+    ModalPaymentDispatcher({
+      type: ModalPaymentConstants.SET_MODAL_VIEW,
+      payload: event.target.value,
```

Comment:
Is there a reason why we are hardcoding `event.target.value` instead of hard coding `ModalPaymentTypes.modalPages.light_up_chinatown` like in the other payment modals? Unless we plan to have other buttons on the same page

Reply:
yeah i think there are different tier buttons that enable different functionality on the modals